### PR TITLE
Extend paramters to include regular sealing sectors in addition to snap sectors

### LINKED ISSUE #10: Adding miner logic
How hard would it be to add some lotus-miner logic to this?
I've been looking for the boost graphql query to incorporate logic around the amount of sectors actively in the sealing pipeline. With that ace-of-spades could get a parameter to only pull in new deals if the miner is below a certain threshold.
Yet, I couldn't work out the query...

### PR DESCRIPTION
**Pull request by jcrowthe**
This PR swaps: `--maximum-snap-sectors-in-flight` for `--maximum-sectors-in-sealing-pipeline`. This new parameter takes into account both snap sectors and regular sealing sectors. 

Fixes #10

### CODE REVIEW

**Comment by bobdubois** [comment]
I think only looking at the current PreCommit1 value is not enough. The script now keeps on loading new deals because of the big delay there is before they land in PC1 state.

I'd suggest to make the sum of:

 {IsOffline: true, Checkpoint: Accepted}
{Checkpoint: Published}
{Checkpoint: Transferred}
{Checkpoint: PublishConfirmed}
and the actual PC1 jobs

**Comment by jcrowthe** [comment]
I tried the sum approach originally and it does not work well. It takes too much into account and is inefficient with what sealing resources you have. Instead, you should optimize your sealing pipeline based on PC1, then ensure the rest of the pipeline has the capacity needed so PC1 is the limiting factor.

When it comes to deals then, there are other limit parameters in Ace allowing you to manage your "buffer" of deals that are coming in so that you always have deals ready to be processed the moment a PC1 processing slot becomes available.

You should b invt nax-dwal-in-flight parameter and the AP1 parameters to manage Ace. Please see the Ace readme for your options. They are necessary to have an efficient pipeline.

tl;dr; your approach was my original design in Ace and it can only produce an inefficient pipeline. Feel free to modify a fork of Ace yourself if you feel otherwise.

**Comment by bobdubois** [comment]
thanks for your elaboration and for you code!
I think I have found a way to keep a healthy, constant pipeline running now.
